sensu-plugins-systemd.gemspec

```diff
@@ -30,17 +33,17 @@ Gem::Specification.new do |s| # rubocop:disable Metrics/BlockLength
   # s.test_files    = Dir['test/*.rb']
   s.test_files    = s.files.grep(%r{^(test|spec|features)/})
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 1.2'
+  s.add_runtime_dependency 'sensu-plugin', '>= 2', '< 5'
   # s.add_dependency('json')
   # s.add_dependency('mixlib-cli', '>= 1.5.0')
 
-  s.add_development_dependency 'bundler',                   '~> 1.7'
+  s.add_development_dependency 'bundler',                   '~> 2.0'
   s.add_development_dependency 'codeclimate-test-reporter', '~> 0.4'
   s.add_development_dependency 'github-markup',             '~> 1.3'
   s.add_development_dependency 'pry',                       '~> 0.10'
   s.add_development_dependency 'rake',                      '~> 10.5'
   s.add_development_dependency 'redcarpet',                 '~> 3.2'
-  s.add_development_dependency 'rubocop',                   '~> 0.37'
   s.add_development_dependency 'rspec',                     '~> 3.4'
+  s.add_development_dependency 'rubocop',                   '~> 0.37'
```

Comment:
oh man thats an old rubocop, gonna make note of that to go back in and look at updating.